### DISCUSSION TITLE
[Feature] Screen Streaming to VSCode for Ev3

### DISCUSSION THREAD

**Discussion by XilefTech** [pybricks/support#223] [Ideas]
Since the screen on the Ev3-Brick has no background light it is hard to notice anything on it in darker environments. It would be cool to have the ability to stream the bricks screen to your computer and display it in VSCode similar to what you can already do with screenshots.

**Comment by dlech** [comment]
This sounds like a feature for https://github.com/ev3dev/vscode-ev3dev-browser which is outside of the scope of Pybricks.

I've also used this project before for streaming: https://github.com/G33kDude/ev3dev-web-remote

**Comment by XilefTech** [comment]
> This sounds like a feature for https://github.com/ev3dev/vscode-ev3dev-browser which is outside of the scope of Pybricks.

okay, I'll maybe add an issue over there but it seems like the other feature requests haven't been worked on for more than a year for some. So it probably will never get implemented, even if I would open an issue in that repo.

> I've also used this project before for streaming: https://github.com/G33kDude/ev3dev-web-remote

looked into that a bit but couldn't get it to run with the given install instructions on pybricks

I guess I'll use the opportunity and implement something like that into charlieOSX myself
